powerauth-nextstep-client/src/main/java/io/getlime/security/powerauth/lib/nextstep/client/NextStepClient.java

```diff
@@ -2707,13 +2710,37 @@ private void logError(NextStepClientException ex) {
         }
     }
 
+    private static NextStepClientException createNextStepClientException(final RestClientException e) {
+        return new NextStepClientException(e, resolveNextStepError(e));
+    }
+
+    private static NextStepError resolveNextStepError(final RestClientException e) {
+        // TODO (racansky, 2022-12-06) workaround until https://github.com/wultra/lime-java-core/issues/32
+        if (e.getErrorResponse() == null || e.getErrorResponse().getResponseObject() == null) {
```

Comment:
Duplicate code to: https://github.com/wultra/powerauth-webflow/blob/3c0a09da6a6f02ea1ade00c37c8b85625c85d460/powerauth-nextstep-client/src/main/java/io/getlime/security/powerauth/lib/nextstep/client/NextStepClientException.java#L121